### PR TITLE
Add SF measurement for 2016 and 2017 UL datasets

### PR DESCRIPTION
**Pull request by mburkart**

### CODE REVIEW

File: merge_outputs.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        sample="Muon"
+        ;;
+
+    *)
+        echo "[ERROR] Given channel ${CHANNEL} not implemented"
+        exit 42
+esac
+
+
+basedir=/store/user/mburkart/CROWN/ntuples/${NTUPLETAG}/CROWNRun/${ERA}/
```
Comment (harrypuuter) [inline merge_outputs.sh]: change these to user instead of mburkart

File: merge_outputs.sh [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+gridpath=root://cmsxrootd-kit.gridka.de:1094/
+output_dir=/storage/9/mburkart/Run2UltraLegacy/scalefactors/${NTUPLETAG}/${ERA}UL
```
Comment (harrypuuter) [inline merge_outputs.sh]: ceph !

File: set_inputfiles.yaml [hunk 3]
```diff
@@ -76,6 +76,77 @@
...
+    # DataF: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2017UL/DoubleMuon_Run2017F_2017UL.root"
+
+  muon:
+    folder: "ntuple"
+    # Data: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2017UL/SingleMuon_2017UL.root"
+    # Embedding: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2017UL/MuonEmbedding_2017UL.root"
+    # DY: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2022_02_13_TandP/2017UL/DYJetsToLL_2017UL_mm.root"
+    Data: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2023_03_17_TandP/2017UL/SingleMuon_2017UL.root"
+    Embedding: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2023_03_17_TandP/2017UL/MuonEmbedding_2017UL.root"
+    DY: "/storage/9/mburkart/Run2UltraLegacy/scalefactors/2023_03_17_TandP/2017UL/DYJetsToLL_2017UL_mm.root"
```
Comment (harrypuuter) [inline set_inputfiles.yaml:139]: should i copy these files to my folders ?

**Review by harrypuuter — APPROVED** [review]

File: set_inputfiles.yaml (hunk 3 above)
Comment (mburkart) [reply set_inputfiles.yaml:139]: Yes, could make sense to do so.

File: merge_outputs.sh (hunk 1 above)
Comment (mburkart) [reply merge_outputs.sh]: Fixed

File: merge_outputs.sh (hunk 2 above)
Comment (mburkart) [reply merge_outputs.sh]: Updated.

**Pull request merged**